components/camel-hbase/pom.xml

```diff
@@ -208,36 +75,18 @@
                     <groupId>log4j</groupId>
                     <artifactId>log4j</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>junit</groupId>
-                    <artifactId>junit</artifactId>
-                </exclusion>
             </exclusions>
         </dependency>
-        <!-- because hbase-server 1.1.1 use hadoop2.5.1 by default, check is it still required by the next version update -->
         <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-client</artifactId>
-            <version>${hadoop2-version}</version>
```

Comment:
I have a question, why you needed to replace the hadoop deps of the hbase client?